newIDE/app/src/UI/Notification/NotificationListItem.js

```diff
@@ -76,6 +78,35 @@ const getNotificationPrimaryTextByType = (
       </Trans>
     );
   }
+  if (notification.type === 'game-sessions-achievement') {
+    if (notification.data.gameCount === 1) {
+      if (notification.data.gameId && notification.data.gameName) {
+        return (
+          <Trans>
+            Your game {notification.data.gameName} was played more than
+            {notification.data.sessionsCount} times!
+          </Trans>
+        );
+      } else return null; // should not happen.
+    }
+    if (notification.data.allGames) {
+      return (
+        <Trans>
+          All your games were played more than {notification.data.sessionsCount}
+          times in total!
+        </Trans>
+      );
+    }
+    const { gameCount } = notification.data;
+    if (Number.isInteger(gameCount)) {
+      return (
+        <Trans>
+          {gameCount} of your games were played more than
+          {notification.data.sessionsCount} times in total!
```

Comment:
Is this "each" or "in total"?

```suggestion
          {gameCount} of your games were played more than
          {notification.data.sessionsCount} times each!
```

Reply:
It's in total: aggregated across the games